### PR TITLE
2 0 x.16.markup support

### PR DESCRIPTION
**Pull request by ericCampos**
Added Native Branding
Added Markdown support

Issues Fixed:
#61 
#37

### CODE REVIEW

**Comment by bart-at-mobicage** [comment]
the parser uses the system font and fontsizes by default.

Let's subclass TSMarkdownParser and override the 
- defaultAttributes
- headerAttributes
- listAttributes
- quoteAttributes
- monospaceAttributes
- strongAttributes
- emphasisAttributes

**Comment by ericCampos** [comment]
The size of the text with markdown is fixed, but the markdown does not work in the "About" of the services.

File: src/MCTMessageDetailView.m [hunk 1]
```diff
@@ -588,11 +589,13 @@ - (UIView *)createMessageViewWithYPosition:(CGFloat)y
     if (!showBranded) {
         int textViewPadding = 9;
 
-        // Calculate height of message
+        self.message.message = @"# H1\n## H2\n### H3\n#### H4\n##### 5\n###### 6";
```
Comment (RubenMattan) [inline src/MCTMessageDetailView.m]: debug code

Comment (bart-at-mobicage) [inline src/MCTMessageDetailView.m]: woops

**Pull request merged**